msteams-platform/concepts/deploy-and-publish/apps-upload.md

```diff
@@ -6,106 +6,113 @@ ms.author: lajanuar
+   The uploaded package is available for use in the team or conversation specified in the consent dialog. 
+   > [!NOTE]
+   >If your app does not appear, the most common reason is an error in the manifest, particularly IDs for the app, bot, and messaging extensions.<br/>
+   If the app is not scoped for conversations, the **Add to a chat** option does not appear.
+
+## Upload your package into a team using the Apps tab
 
 >[!NOTE]
-> Apps in conversations is currently in [Developer Preview](../../resources/dev-preview/developer-preview-intro.md), and the option does not appear if Teams is not running in that mode.
+> **Apps** tab in conversations is currently in [Developer Preview](../../resources/dev-preview/developer-preview-intro.md), and the option does not appear if Teams is not running in that mode.
 
 ![Example of bot in list of uploaded bots](../../assets/images/botinlist.jpg)
 
-## Upload your package into a team using the Apps tab
+You can also upload your app package using the **Apps** tab.
+
+**To upload your app package**
 
 1. In the target team, choose **More options** (**&#8943;**) and select **Manage team**.
 
    > [!NOTE]
    > You must be the team owner or the owner must give access to users to add the appropriate app types for this functionality to appear.
 
-2. Select the **Apps** tab and choose **Upload a custom app** on the lower right.
+2. Select the **Apps** tab, and then choose **Upload a custom app** on the lower right.
 
    ![Upload entry point](../../assets/images/UploadACustomApp.png)
 
-3. Select your .zip package from the computer.
-
-4. You can see your uploaded app in the list.
+3. In the **Open** dialog, navigate to your .zip package, and then select it.
+  
+4. Select **Open** and your uploaded app appears in the list.
 
    ![Example of bot in list of uploaded bots](../../assets/images/botinlist.jpg)
 
-If your app does not load, the most common reason is an error in the manifest, particularly IDs for the app, bot, and messaging extensions.
+   > [!NOTE]
+   > If your app does not appear, the most common reason is an error in the manifest, particularly IDs for the app, bot, and messaging extensions.
 
-## Access your uploaded configurable tab
+## Troubleshoot
 
-If the app contains tabs, users can pin them to any conversation or team channel using the standard tab gallery flow:
+If the manifest fails to load, check that you followed all the instructions in [create the app package](../../concepts/build-and-test/apps-package.md) and validated your manifest against the [schema](../../resources/schema/manifest-schema.md).
 
-1. Go to a channel in the team. Choose **+** to add a tab to the right of the existing tabs.
 
-2. Select your tab from the gallery that appears.
+## Test the uploaded app package
```

Comment:
@KirtiPereira I think you can remove Lines 76-136. We don't need to cover how to test/add an app here since that is covered in end user docs and could change frequently, making this info stale.